### PR TITLE
update the log error messages and execute of cmd and output

### PR DESCRIPTION
**Pull request by kobihk**
1. when we exit with sys.exit(1) the line before should be log.error instead of log.info
2. format with f-string which is more readable
3. in the method execute_command(cmd) log the cmd and the output instead of doing that multiple times in the code

I faced these issues as part of jira ticket: RHOAIENG-2683

### CODE REVIEW

**Review by aloganat — DISMISSED** [review]

File: ods_ci/utils/scripts/ocm/ocm.py [hunk 1]
```diff
@@ -733,7 +720,6 @@ def create_idp(self):
 
             # time.sleep(10)
 
-            # log.info("CMD: {}".format(cmd))
```
[the remark this answers is not in the document]
Comment (kobihk) [reply ods_ci/utils/scripts/ocm/ocm.py:736]: The log of CMD would be part of the method execute_command(cmd) so we don't need this code in any way :)

**Review by aloganat — APPROVED** [review]

File: ods_ci/utils/scripts/util.py [hunk 2]
```diff
@@ -71,10 +73,10 @@ def execute_command(cmd):
                 line = p.stdout.readline()
                 if line != "":
                     output += line + "\n"
-                    print(line)
```
[the remark this answers is not in the document]
Comment (jiridanek) [reply ods_ci/utils/scripts/util.py:74]: clone_config_repo does not sound particularly longrunning, thankfully

[the remark this answers is not in the document]
Comment (jstourac) [reply ods_ci/utils/scripts/util.py:74]: but this change is part of the `def execute_command(cmd):` method... :wink:

[the remark this answers is not in the document]
Comment (kobihk) [reply ods_ci/utils/scripts/util.py:74]: @jstourac The while loop here is just to add "\n" at the end of each line after the command is completed.
I ran with this fix many times and didn't face any issues so let's give it a chance and if we see any issue as you described we always can fix that ;-)

[the remark this answers is not in the document]
Comment (jstourac) [reply ods_ci/utils/scripts/util.py:74]: I have to admit that I know almost nothing about Python, but from what I tired, the while loop isn't called after the command is completed but during the command is running. Try to play e.g. with something like this:

```
import subprocess
import sys

cmd = "while true; do echo 'ahoj'; sleep 2; done"
#cmd = "echo ahoj; sleep 2"
output = ""

print(f"CMD: {cmd}")

with subprocess.Popen(
    cmd,
    shell=True,
    stdout=subprocess.PIPE,
    stderr=subprocess.STDOUT,
    universal_newlines=True,
    encoding="utf-8",
    errors="replace",
) as p:
    while True:
        line = p.stdout.readline()
        if line != "":
            output += line + "\n"
            print(line)
        elif p.poll() is not None:
            break
    sys.stdout.flush()
    print(f"OUTPUT: {output}")
```

Thus as I said - we don't have to face any issue for a short running calls here, but we may eventually run into issue of missing a long-running command execution that has been terminated before the command ended and could print out the final `output`. Also, with this change we don't have any idea what is happening in the meantime for these long-running commands.

[the remark this answers is not in the document]
Comment (jiridanek) [reply ods_ci/utils/scripts/util.py:74]: I agree there's something to improve, I'm opening a new ticket to track

* https://github.com/red-hat-data-services/ods-ci/issues/1241

[the remark this answers is not in the document]
Comment (kobihk) [reply ods_ci/utils/scripts/util.py:74]: great code example to approve your point 👍🏽  
@jstourac @jiridanek you are right, I missed that.
Thank you!!

**Pull request merged**